hazelcast/src/main/java/com/hazelcast/client/impl/spi/ClientContext.java

```diff
@@ -170,7 +171,7 @@ public ClientTransactionManagerService getTransactionManager() {
         return transactionManager;
     }
 
-    public ClientExecutionService getExecutionService() {
+    public TaskScheduler getExecutionService() {
```

Comment:
I think we can remove `ClientExecutionService` interface completely and also change the method name to `getTaskScheduler` . There seems to be no use left for ClientExecutionService interface anymore.